src/commands/packageModule.ts

```diff
@@ -35,7 +35,16 @@ export async function packageModule (node: DeviceNode | OSVerNode | PlatformNode
 		await vscode.tasks.executeTask(task);
 	} catch (error) {
 		if (error instanceof InteractionError) {
-			await handleInteractionError(error);
+			await handleInteractionError(error)
+				.then(async function () {
+					const choice = await vscode.window.showErrorMessage('Package Module failed', { title: 'Rerun' });
+					if (!choice) {
+						return;
+					}
+					if (choice.title === 'Rerun') {
```

Comment:
<!--
  0 failure: 
  1 warning:  src/commands/pack...
  
  
  DangerID: danger-id-Danger;
  File: src/commands/packageModule.ts;
  Line: 44;
-->

- :warning: src/commands/packageModule.ts line 44 – Each then() should return a value or throw (promise/always-return)